secretsauce/apps/account/models.py

```diff
@@ -69,15 +55,12 @@ class User(AbstractUser):
     company = models.ForeignKey(
         Company,
         on_delete=models.CASCADE,
-    )
-    invitation = models.OneToOneField(
-        Invitation,
-        on_delete=models.CASCADE,
+        null=True
```

Comment:
Should a User be able to not belong to a company?

Reply:
Wanted to make it more flexible but on hindsight, I think it's best to set some rules for Users. Will change it back to have to include company.